src/wallet/crypter.h

```diff
@@ -230,6 +232,10 @@ class CCryptoKeyStore : public CBasicKeyStore
             mi++;
         }
     }
+    //! Sapling 
+    virtual bool AddCryptedSaplingSpendingKey(const libzcash::SaplingFullViewingKey &fvk,
+                                       const std::vector<unsigned char> &vchCryptedSecret);
+    bool AddSaplingSpendingKey(const libzcash::SaplingSpendingKey &sk);
```

Comment:
Todo in subsequent PR: implement crypter overrides of `HaveSaplingSpendingKey()` etc.